src/router/index.ts

```diff
@@ -142,31 +142,10 @@ export function createWorkspaceRouteRecords(components: Partial<WorkspaceRouteCo
         can: 'read:work_queue',
       },
       children: [
-        {
-          name: 'workspace.work-queues',
-          path: '',
-          component: components.workQueues,
-        },
-        {
-          name: 'workspace.work-queues.work-queue-create',
-          path: 'create',
-          component: components.workQueueCreate,
-          meta: {
-            can: 'create:work_queue',
-          },
-        },
         {
           name: 'workspace.work-queues.work-queue',
           path: 'work-queue/:workQueueId',
```

Comment:
Going to hold onto this route for now. There are 2 components that use it so for now this route renders a stub that redirects to the newer work-pool.work-queue route.